README.md

```diff
@@ -516,7 +519,10 @@ const customCancellationReasons = [
 ];
 
 <Provider store={store}>
-  <PlanDetails customCancellationReasons={customCancellationReasons} />
```

Comment:
what about Subscriptions component? Does this component include the Unsubscribe flow?

Reply:
I will check that